package.json

```diff
@@ -28,6 +28,8 @@
   "main": "./lib/azure.js",
   "license": "(MIT OR Apache-2.0)",
   "dependencies": {
+    "@microsoft.azure/autorest.modeler": "^2.0.18",
+    "@microsoft.azure/autorest.nodejs": "^2.0.16",
```

Comment:
Why do we need the above 2 as a dependency? Please remove them.

Reply:
removed